docs/photoniq/event-delivery/event-delivery-filters.md

```diff
@@ -0,0 +1,165 @@
+
+:::
+
+<Tabs groupId="operating-systems">
+  <TabItem value="ws-syntax" label="WebSockets">
+
+Curl currently has no support for WebSockets, you can use [wscat](https://github.com/WebSockets/wscat) to send a request to the [Subscribe to Stream API](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get) via WebSockets as shown below:
+
+```bash
+  wscat -c 'wss://<eds-host>/api/es/v1/subscribe?type=collection&x-customer-id=<x-customer-id>&filters={"action": "add", "once": "FALSE", "queries": ["select fieldName from CollectionName"]}'
+```
+
+  </TabItem>
+  <TabItem value="sse-syntax" label="Server-Sent Events">
+
+Here is the syntax to send a request to the [Subscribe to Stream API](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get) via SSE:
+
+```bash
+  curl -X POST -H "Content-Type: application/json" -H "x-customer-id: <x-customer-id>" -d '{"type": "collection", "filters": {"once": "TRUE", "compress": "FALSE", "initialData":"TRUE", "queries": ["select fieldName from CollectionName"]}}' https://<eds-host>/api/es/sse/v1/subscribe
+```
+  </TabItem>
+</Tabs>
+
+
+
+**Subscribe to specific field changes**
+
+Subscribing to specific field changes is useful when monitoring a specific attribute of a dataset for changes — such as price adjustments in a product catalog or status updates in a task tracking system. For example, an e-commerce platform might use this to update the UI in real-time when a product price changes, enhancing the customer experience.
+
+<Tabs groupId="operating-systems">
+  <TabItem value="ws-specific" label="WebSockets">
+
+```bash
+  wscat -c 'wss://<eds-host>/api/es/v1/subscribe?type=collection&x-customer-id=<x-customer-id>&filters={"action": "add", "once": "FALSE", "queries": ["select fieldName from CollectionName"]}'
+```
+
+  </TabItem>
+  <TabItem value="sse-specific" label="Server-Sent Events">
+
+
+```bash
+  curl -X POST -H "Content-Type: application/json" -H "x-customer-id: <x-customer-id>" -d '{"type": "collection", "filters": {"once": "TRUE", "queries": ["select fieldName from CollectionName"]}}' https://<eds-host>/api/es/sse/v1/subscribe
+```
+  </TabItem>
+</Tabs>
+
+The above command subscribes to changes in `fieldName` within `CollectionName`. 
+
+**Subscribe to all events in a Collection**
+
+This is particularly beneficial for applications that need to maintain a real-time view of an entire dataset, such as a dashboard displaying the latest metrics across various data points or a live feed of social media posts. For example, a monitoring tool could use this to alert operators of any changes across all monitored systems, ensuring swift response to incidents.
+
+<Tabs groupId="operating-systems">
+  <TabItem value="ws-all" label="WebSockets">
+
+```bash
+  wscat -c 'wss://<eds-host>/api/es/v1/subscribe?type=collection&x-customer-id=<x-customer-id>&filters={"action": "add", "once": "FALSE", "queries": ["select fieldName from CollectionName"]}'
```

Comment:
Did you intend to say - select * from CollectionName here instead of fieldName, since you want all updates from the collection ?